lib/sifter.ts

```diff
@@ -353,3 +353,4 @@ class Sifter{
 }
 
 export { Sifter, scoreValue, getAttr, getAttrNesting, propToArray, iterate, cmp, getPattern }
+export * from './types.ts';
```

Comment:
I noticed that `tom-select` was reaching into `dist/types/...` to get these; figured it'd be cleaner to export them from the top-level module.